yogstation/code/modules/clothing/donator.dm

```diff
@@ -189,14 +189,26 @@
 	name = "Inmate belongings box"
 	desc = "A criminals' belongings, returned upon their release. Just stop killing people."
 
-/obj/item/storage/box/boxta/PopulateContents() 
+/obj/item/storage/box/boxta/PopulateContents()
 	. = ..()
 	new /obj/item/clothing/neck/falcon(src)
 	new /obj/item/clothing/head/fedora/gtrim_fedora(src)
 
+/obj/item/storage/box/somecustodian
+	name = "Golden Box"
+	desc = "Holds a set of one of the most powerful suits of armor known to mankind.(99 cents rip off)"
+
+/obj/item/storage/box/somecustodian/PopulateContents()
+	. = ..()
+	new /obj/item/clothing/head/custodianhelm(src)
+	new /obj/item/clothing/suit/custodianchest(src)
+
+/obj/item/bedsheets/pridebedsheet/cark
```

Comment:
these changes are not a part of your pr

Reply:
top part is but I was just trying to fix my mistake

Reply:
remove the `/obj/item/bedsheets/pridebedsheet/cark` line